ci-operator/config/openshift/osin/openshift-osin-master.yaml

```diff
@@ -17,7 +17,7 @@ resources:
     requests:
       cpu: "3"
       memory: 8Gi
-test_binary_build_commands: go build -race ./...
+test_binary_build_commands: go build -race -mod=mod ./...
```

Comment:
openshift builds require offline builds and vendor folder. openshift/osin needs a bump commit

Reply:
This repository is not being built offline, it has never had a vendor/ folder. I am merely using openshift CI for the community contributions to it.